### ISSUE TITLE
A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_Screenreader: Voiceover announces as ‘Empty Table’ when navigated to search section’s table.

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9291]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOS;#MAS1.3.1;#Screenreader;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3
Additional Details:
Mac OS:10.15.3
MAS References: MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Connect to server.
3. In the Connections pane, right-click on a server & select Manage. 
4. Launch voiceover
5. Navigate to Search section’s table present below editbox using Control+ Option+ arrow key

**Actual:**
While Voiceover is on and navigated to Search section’s table which is present below editbox using Control+ Option + arrow key, voiceover announce as Empty table.

**Same issue is observed in Database Dashboard screen:** 1. Launch Azure Data Studio.2. Connect to server.3. In the Connections pane, navigate to server --> database --> right click on any database & select Manage.

**Expected:**
While Voiceover is on and navigated to Search section’s table which is present below editbox using Control+ Option + arrow key, voiceover should announce in the format ‘Table_Name Table’.

**User Impact:**
Screen reader users will think as Search section’s table is not having any data.

**Attachment link for Reference:**
[9291_A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4274828/9291_A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_Screenreader.zip)

**Comment by alanrenmsft** [comment]
it is a tree control, not sure why is it reading it as table

**Comment by alanrenmsft** [comment]
test with other screen readers to see whether it is reading the widget as tree and announce the item count. close as external voice over issue if works fine with other screen readers.

**Comment by alanrenmsft** [comment]
this worked well with narrator and seems to be voiceOver issue, we are using the vscode tree UI element and they also have the same problem: https://github.com/microsoft/vscode/issues/40601

closing as an external issue

**Comment by sspatil123** [comment]
GithubTags:#Closed; Version 1.41.1
![image](https://user-images.githubusercontent.com/85863999/216964036-d8a71ce1-2353-4637-9b4f-219e7062b751.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
